proto/meta.proto

```diff
@@ -345,7 +350,10 @@ message AlterConnectorPropsRequest {
   map<string, string> changed_props = 2;
   map<string, secret.SecretRef> changed_secret_refs = 3;
   optional uint32 connector_conn_ref = 4;
-  AlterConnectorPropsObject object_type = 5;
+  oneof object_type {
+    AlterConnectorPropsObject alter_connector_props_object = 6;
+    AlterIcebergTablePropsObject alter_iceberg_table_props_object = 7;
+  }
```

Comment:
I think we should keep the original `object_type` as is and introduce an additional optional field to pass the iceberg sink and source ids.